#Fri Sep 25 20:28:43 CEST 2020
admin.deleteUser.intro=Deleta todos os usu\u00E1rios que tenham sido removidos do diret\u00F3rio LDAP
admin.deleteUser.title=Remove os usu\u00E1rios deletados no LDAP
admin.logview.title=Acessando
admin.menu.ldap=LDAP
admin.menu.ldap.desc=Administra\u00E7\u00E3o da sincroniza\u00E7\u00E3o LDAP
admin.synchronize.finished.failure=sincroniza\u00E7\u00E3o LDAP terminada. Os seguintes erros ocorreram\: (0)
admin.synchronize.finished.success=sincroniza\u00E7\u00E3o LDAP finalizada com sucesso
admin.synchronize.intro=Pressione "iniciar" para iniciar imediatamente uma sincroniza\u00E7\u00E3o LDAP. Seja paciente, esta a\u00E7\u00E3o pode demorar um pouco para terminar.
admin.synchronize.last=\u00DAltima sincroniza\u00E7\u00E3o
admin.synchronize.last.unknown=desconhecido
admin.synchronize.started=Sincroniza\u00E7\u00E3o LDAP iniciada. Favor aguarde...
admin.synchronize.title=Sincroniza\u00E7\u00E3o
admin.title=Administra\u00E7\u00E3o LDAP
authentication.provider.description=Voc\u00EA tem uma conta no servidor LDAP central?
authentication.provider.linkText=Entrar com conta LDAP
checkall=Seleciona tudo
delete.button.start=Inicia
delete.error.connection=Ocorreu um erro durante a conex\u00E3o com o servidor LDAP. Por favor, verifique suas configura\u00E7\u00F5es LDAP e tente novamente mais tarde.
delete.error.connection.close=A conex\u00E3o LDAP n\u00E3o pode ser fechado ap\u00F3s o processo de sincroniza\u00E7\u00E3o manual.
delete.error.lock=Outro trabalho j\u00E1 est\u00E1 rodando. Favor tente novamente mais tarde
delete.setp1.content.confirm=Os seguintes usu\u00E1rios ser\u00E3o removidos.
delete.step.cancel=Cancela, nenhum usu\u00E1rio deletado\!
delete.step.finish.noUsers=Nenhum Usu\u00E1rio Deletado\!
delete.step.finish.users={0} User\u00E1rios Deletados\!
delete.step.noUsers=Nenhum Usu\u00E1rio encontrado para remo\u00E7\u00E3o\!
delete.step0.content=Seleciona Usu\u00E1rios a serem deletados
delete.step0.description=Selecione o usu\u00E1rio a deletar
delete.step1.content.nothingToDelete=Nenhum usu\u00E1rio selecionado para remo\u00E7\u00E3o
delete.step1.description=Confirma
error.password.change.not.allow=Voc\u00EA est\u00E1 logado com um acesso LDAP no OpenOlat. Para alterar sua senha, entre em contato com o administrador do sistema.
error.user.already.in.use=Este nome de usu\u00E1rio j\u00E1 \u00E9 usado por outro usu\u00E1rio OpenOlat.
error.user.not.found=O usu\u00E1rio n\u00E3o pode ser encontrado
ldap.maintenance=Manuten\u00E7\u00E3o
lf.error.loginempty=Favor insira seu usu\u00E1rio LDAP
lf.error.passempty=Favor insira o seu password LDAP
lf.login=Usu\u00E1rio
lf.pass=Senha
login.error=<b>Usu\u00E1rio ou senha LDAP inv\u00E1lido.</b><p>Raz\u00F5es poss\u00EDveis podem ser\:<ul><li><b>Voc\u00EA digitou errado\:</b><br>Favor tente novamente\!<li><b>Usu\u00E1rio ou senha errados\!</b><br /><br />LDAP ERROR\: {0}</ul></p>
login.form=Login OLAT via LDAP
login.intro=Favor acesse com seu usu\u00E1rio e senha LDAP.
login.notauthenticated=<b>N\u00E3o autenticado para o OLAT</b><p>Voc\u00EA n\u00E3o tem permiss\u00E3o para efetuar login no OLAT.</p><p>Contate o administrador de usu\u00E1rios para ter acesso.</p>
one.user.sync.button.start=Sincronizar conta de usu\u00E1rio
remove.fallback.auth=Excluir cache fallback de autentica\u00E7\u00F5es
remove.fallback.auth.info=Ao retirar as autentica\u00E7\u00F5es fallback, um login n\u00E3o \u00E9 mais poss\u00EDvel se a conex\u00E3o com o servidor LDAP falhar. O cache funciona novamente ap\u00F3s um login bem-sucedido.
sync.button.start=Iniciar sincroniza\u00E7\u00E3o
uncheckall=Deseleciona tudo
